Userland/Utilities/js.cpp

```diff
@@ -837,7 +837,10 @@ static bool parse_and_run(JS::Interpreter& interpreter, StringView source, Strin
 
             if (s_run_bytecode) {
                 JS::Bytecode::Interpreter bytecode_interpreter(interpreter.global_object(), interpreter.realm());
-                TRY_OR_DISCARD(bytecode_interpreter.run(executable));
+                auto result = bytecode_interpreter.run(executable);
+                // Until the AST interpreter gives a completion we just pretend a exception was thrown.
```

Comment:
AST interpreter? This is bytecode... also `a exception` -> `an exception`.

Reply:
I meant that the AST interpreter does not give a ThrowCompletionOr<> yet. But yes, this comment isn't clear.

Reply:
ah, I see what you mean now :)
how about:

```suggestion
                // Until the AST interpreter also returns a completion, we have to turn the completion here into an exception stored on the VM, so that the exception check further below works in both cases.
```